### ISSUE TITLE
Textures glitched and blinking in ctlava world

### ISSUE THREAD

**Issue by Anglodel** [securas/SealedBite#10]
Sorry I send it too quickly. I’m editing the issue.

**Comment by securas** [comment]
Thanks for reporting the issue. However, it's an engine problem. To my misfortune, there's not much I can do about it.

**Comment by asheraryam** [comment]
@securas There is a workaround in the settings I think called "nvidia flicker" something, maybe it's worth checking out.

There's also this issue. https://github.com/godotengine/godot/issues/9913

Also might be worth trying godot 3.2 with the gles2 and batching improvements.

**Comment by securas** [comment]
@asheraryam This game is and will remain as a 3.1.2 game. It is not compatible with 3.2.x for a number of reasons. Having said that, it is possible that someone took the time to fork and port it to 3.2 Nevertheless, this will remains as the original. As such, the nvidia flicker something that you've mentioned is only supported for GLES2, which does not support the GPU particles that I use. Finally, the nvidia flicker thing is not necessarily solving this problem, which it seems to be a separate issue related to drawing polygons. I have an nvidia card and never saw this problem. In fact, @Anglodel did not specify if the game was running on nvidia or any other gpu.

**Comment by asheraryam** [comment]
Okay thanks for clarifying :D I have no idea what could be causing this other than driver issues since I haven't seen such a problem before, so until it's updated to 3.2.2 there's nothing for us to do about it.

**Comment by Anglodel** [comment]
> 
> 
> @asheraryam This game is and will remain as a 3.1.2 game. It is not compatible with 3.2.x for a number of reasons. Having said that, it is possible that someone took the time to fork and port it to 3.2 Nevertheless, this will remains as the original. As such, the nvidia flicker something that you've mentioned is only supported for GLES2, which does not support the GPU particles that I use. Finally, the nvidia flicker thing is not necessarily solving this problem, which it seems to be a separate issue related to drawing polygons. I have an nvidia card and never saw this problem. In fact, @Anglodel did not specify if the game was running on nvidia or any other gpu.

Indeed, I just have a integrated graphics chipset, with my Intel CPU, an Intel UHD Graphics 605…
So maybe with a driver update, my problem could be fixed ?
